modules/ROOT/pages/to-find-info.adoc

```diff
@@ -68,15 +68,56 @@ Keyword or term that describes a specific search.
 +
 Lifecycle state can be stable, deprecated, or development.
 
-Filter options appear when they are available within an organization. 
+* Custom field
++
+Custom fields are specific metadata fields set by the administrator of a business group or organization owner.
+
+Filter options appear when they are available within an organization.
 
-To filter search results:
+=== Filter By Type, Organization, Category, and Lifecycle State
 
-. From the Exchange landing page, select a filter option.
+. In the Exchange landing page, select a filter option.
+. To find an option from a long list, use the scroll bar or enter a search term.
 . Click one or more options from the menu and click *Apply*. 
 . To clear a filter option, click *Clear filter*.
 . To clear all filter options, click *Clear all filters*.
 
+=== Filter By Custom Fields
+Custom fields are created by Exchange administrators and organization owners using the Exchange API to extend the Exchange asset metadata model and add custom data to any type of asset. The custom field filter lets you refine a search using preset information.
+
+
+The custom field filter has the following data types:
+
+[%header,cols="1a,1a,2a"]
+|===
+|Data Type |Example Name for Data Type|Operator
```

Comment:
In my opinion, the `Example name for data type` does not add useful information here